package.json

```diff
@@ -26,7 +26,7 @@
     "@babel/runtime": "7.10.1",
     "bluebird": "2.9.25",
     "core-js": "3.6.5",
-    "forest-express": "7.7.2",
+    "forest-express": "7.9.1",
```

Comment:
No update for the `yarn.lock` file?

Reply:
I will have too, once `forest-express` 7.9.1 is released :)